proto/osmosis/concentrated-liquidity/query.proto

```diff
@@ -108,6 +108,13 @@ service Query {
                                    "cfmm_pool_id_link_from_concentrated/"
                                    "{concentrated_pool_id}";
   }
+
+  rpc UserUnbondingPositions(UserUnbondingPositionsRequest)
+      returns (UserUnbondingPositionsResponse) {
+    option (google.api.http).get = "/osmosis/concentratedliquidity/v1beta1/"
+                                   "user_unbonding_positions/"
+                                   "{address}";
```

Comment:
AFAIR, specifying this on separate lines causes issues. CC: @mattverse for confirmation

Reply:
There is no way to do that with the linter set right?

Reply:
Jon has been utilizing this endpoint locally, so I am going to assume its good to go. Going to merge this to reduce overhead of open PRs